app/BeaconBridge/Properties/launchSettings.json

```diff
@@ -0,0 +1,41 @@
+﻿{
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:62707",
+      "sslPort": 44310
+    }
+  },
```

Comment:
```suggestion
```
Don't need the IIS profile; in fact I'd prefer not to have it and naturally have all users run Kestrel